### PR TITLE
Convert device notes to blade

### PR DESCRIPTION
**Pull request by Jellyfrog**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: resources/views/device/tabs/notes.blade.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@extends('device.index')
+
+@section('tab')
+    <x-panel title="{{ __('Device Notes') }}">
```
Comment (murrant) [inline resources/views/device/tabs/notes.blade.php:4]: @lang() blade directive is preferred

**Comment by murrant** [comment]
Just one style change and this looks good to me.  Nice ;)

File: resources/views/device/tabs/notes.blade.php (hunk 1 above)
Comment (Jellyfrog) [reply resources/views/device/tabs/notes.blade.php:4]: It doesn't work because `title` is a variable, so it returns `Call to undefined function lang()`

Comment (Jellyfrog) [reply resources/views/device/tabs/notes.blade.php:4]: Pick your poison:  
`title="@lang('Device Notes')"` = `@lang('Device Notes')`  
`title="{{ @lang('Device Notes') }}"` = `Call to undefined function lang()`  
`:title="@lang('Device Notes')"` = `Call to undefined function lang()`  
`title="{{ __('Device Notes') }}` = `Device Notes`

Comment (murrant) [reply resources/views/device/tabs/notes.blade.php:4]: Huh, components must be applied first.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-66-release-changelog-july-2020/12953/1

**Pull request merged**